style.css

```diff
@@ -113,7 +113,282 @@ html {
   color: rgba(80, 95, 121, 1);
 }
 
-.social-content {
+.works {
+  background-color: #f7f7f9;
+  margin: 0;
+  display: grid;
+  grid-template-rows: auto;
+  padding-top: 80px;
+}
+
+.work-flex {
+  height: auto;
+  background: #ebebff;
+  margin: 0 15px 80px 15px;
+  border-style: solid;
+  border-color: rgb(219, 214, 214);
+  border-width: 1px;
+  border-radius: 15px;
+  display: flex;
+  flex-direction: column;
+}
+
+.works-flex-item {
+  margin: 0 15px 0 15px;
+  justify-content: center;
+}
+
+.work-image {
+  align-self: center;
+  border-radius: 10px;
+  margin-top: 15px;
+}
+
+.intro-works-text {
+  font-weight: 650;
+  padding-top: 10px;
+  font-size: 35px;
+  font-family: 'Poppins', sans-serif;
+  color: #344563;
+  margin: 0 0 0 0;
+}
+
+.titles-container {
+  display: flex;
+  flex-direction: row;
+  padding-left: 0;
+  justify-self: flex-start;
+  padding-top: 10px;
+  width: 100%;
+}
+
+.list-main {
+  list-style-type: none;
+  font-weight: 600;
+  font-family: 'Poppins', sans-serif;
+  font-size: 13px;
+  color: #344563;
+  padding-right: 25px;
+  flex: 1;
+}
+
+.titles-item {
+  font-weight: 500;
+  font-size: 12px;
+  font-family: 'Poppins', sans-serif;
+  color: rgba(122, 134, 154, 1);
+  list-style-image: url("Counterping_1.png");
+  flex: 2;
+}
+
+.works-description {
+  font-weight: normal;
+  font-size: 15px;
+  font-family: 'Poppins', sans-serif;
+  color: #344563;
+  padding-top: 10px;
+  margin-top: 0;
+}
+
+.lang-container {
+  padding-top: 25px;
+  padding-left: 0;
+  width: 70%;
+  margin: 0;
+  display: flex;
+  justify-content: space-around;
+  flex-direction: row;
+  list-style: none;
+}
+
+.lang {
+  font-size: 15px;
+  border-radius: 8px;
+  padding: 4px 12px 4px;
+  align-items: center;
+  background: #ebebff;
+  color: #6070ff;
+}
+
+.btn-see-project {
+  text-decoration: none;
+  color: #4053fc;
+}
```

Comment:
- [ ] In the original design, this button has a border and different typography. You should consider changing your style to match the design better:

Figma design:
![image](https://user-images.githubusercontent.com/66967274/143579763-f5b61c30-9ccc-438b-989f-c10b41eabdce.png)


Button typography: 
![image](https://user-images.githubusercontent.com/66967274/143579692-26d696fd-529e-44f4-ae08-edd13c7d4d20.png)



Your design:
![image](https://user-images.githubusercontent.com/66967274/143579459-196e8d47-e52b-40ec-9cb0-f245842068e9.png)